### DISCUSSION TITLE
Can someone tell me how to use the translation files in the lang folder correctly?

### DISCUSSION THREAD

**Discussion by Sm1rkBoy** [Nereziel/cs2-WeaponPaints#348] [Q&A] [answered]
I'm not sure if this is a bug, but when I set `ServerLanguage` in addons/counterstrikesharp/configs/core.json to `en`, the `!ws` command works fine.

![1729966776980.png](https://www.helloimg.com/i/2024/10/27/671d32821403c.png)

When I set `ServerLanguage` in addons/counterstrikesharp/configs/core.json to `zh-Hans`, the `!ws` command don not working properly.

![1729967169093.png](https://www.helloimg.com/i/2024/10/27/671d340aabb9e.png)

Also there is no error or warning log, I haven't found any doc about it.....

**Comment by mrc4tt** [comment]
You need to translate it into your language.

There is no `zh-Hans.json` only `zh-cn.json`

**Reply by Sm1rkBoy** [reply]
I set `ServerLanguage` in addons/counterstrikesharp/configs/core.json to `zh-cn` and restart my server.

![1729967500714.png](https://www.helloimg.com/i/2024/10/27/671d3556b6107.png)

But it still not working properly,Because of  `zh-cn` setting, the MatchZy plugin that was originally able to display Chinese is no longer able to display Chinese.

![1729967711073.png](https://www.helloimg.com/i/2024/10/27/671d3627a1e9f.png)

**Reply by mrc4tt** [reply]
This plugin has nothing to do with MatchZy.

**Comment by Sm1rkBoy** [comment, marked as answer]
I solved the problem, I set `ServerLanguage` in addons/counterstrikesharp/configs/core.json to `zh-Hans`,then rename the `zh-cn.json` to `zh-Hans.json` and restart the server.It works!!!

![1729969661901.png](https://www.helloimg.com/i/2024/10/27/671d3dc59cd74.png)
